presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClientLocal.java

```diff
@@ -80,4 +80,7 @@ public void testGetAllTableColumnsInSchema() {}
 
     @Override
     public void testGetTableNames() {}
+
+    @Override
+    public void testGetTableSchemaOffline() {}
```

Comment:
Other test cases that use `metadata.listTableColumns` are override to be empty when testing against local client. So this is also added.